### ISSUE TITLE
ssd-large benchmark problem

### ISSUE THREAD

**Issue by sandip761** [mlperf/inference_results_v0.5#41]
[2020-09-03 18:47:27,646 main.py:83 INFO] Building engines for ssd-large benchmark in Offline scenario...
[2020-09-03 18:47:27,651 main.py:93 INFO] Building DLA engine for Xavier_ssd-large_Offline
[2020-09-03 18:47:34,010 builder.py:74 INFO] Using DLA: Core 0
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'models.ssd_r34.SSD_R34' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torch.nn.modules.container.Sequential' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torch.nn.modules.conv.Conv2d' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torch.nn.modules.batchnorm.BatchNorm2d' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torch.nn.modules.activation.ReLU' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torch.nn.modules.pooling.MaxPool2d' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torchvision.models.resnet.BasicBlock' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
/usr/local/lib/python3.6/dist-packages/torch/serialization.py:593: SourceChangeWarning: source code of class 'torch.nn.modules.container.ModuleList' has changed. you can retrieve the original source code by accessing the object's source attribute or set `torch.nn.Module.dump_patches = True` and use the patch tool to revert the changes.
  warnings.warn(msg, SourceChangeWarning)
[2020-09-03 18:47:34,817 builder.py:119 INFO] Building ./build/engines/Xavier/ssd-large/Offline/ssd-large-Offline-dla-b1-int8.plan
[TensorRT] WARNING: Default DLA is enabled but layer (Unnamed Layer* 132) [PluginV2IOExt] is not supported on DLA, falling back to GPU.
[TensorRT] INFO: Reading Calibration Cache for calibrator: EntropyCalibration2
[TensorRT] INFO: Generated calibration scales using calibration cache. Make sure that calibration cache has latest scales.
[TensorRT] INFO: To regenerate calibration cache, please delete the existing one. TensorRT will generate a new calibration cache.
[TensorRT] INFO:
[TensorRT] INFO: --------------- Layers running on DLA:
[TensorRT] INFO: {(Unnamed Layer* 0) [Convolution],(Unnamed Layer* 1) [Scale],(Unnamed Layer* 2) [Activation],(Unnamed Layer* 3) [Pooling],(Unnamed Layer* 4) [Convolution],(Unnamed Layer* 5) [Scale],(Unnamed Layer* 6) [Activation],(Unnamed Layer* 7) [Convolution],(Unnamed Layer* 8) [Scale],(Unnamed Layer* 9) [ElementWise],(Unnamed Layer* 10) [Activation],(Unnamed Layer* 11) [Convolution],(Unnamed Layer* 12) [Scale],(Unnamed Layer* 13) [Activation],(Unnamed Layer* 14) [Convolution],(Unnamed Layer* 15) [Scale],(Unnamed Layer* 16) [ElementWise],(Unnamed Layer* 17) [Activation],(Unnamed Layer* 18) [Convolution],(Unnamed Layer* 19) [Scale],(Unnamed Layer* 20) [Activation],(Unnamed Layer* 21) [Convolution],(Unnamed Layer* 22) [Scale],(Unnamed Layer* 23) [ElementWise],(Unnamed Layer* 24) [Activation],(Unnamed Layer* 25) [Convolution],(Unnamed Layer* 26) [Scale],(Unnamed Layer* 27) [Activation],(Unnamed Layer* 28) [Convolution],(Unnamed Layer* 29) [Scale],(Unnamed Layer* 30) [Convolution],(Unnamed Layer* 31) [Scale],(Unnamed Layer* 32) [ElementWise],(Unnamed Layer* 33) [Activation],(Unnamed Layer* 34) [Convolution],(Unnamed Layer* 35) [Scale],(Unnamed Layer* 36) [Activation],(Unnamed Layer* 37) [Convolution],(Unnamed Layer* 38) [Scale],(Unnamed Layer* 39) [ElementWise],(Unnamed Layer* 40) [Activation],(Unnamed Layer* 41) [Convolution],(Unnamed Layer* 42) [Scale],(Unnamed Layer* 43) [Activation],(Unnamed Layer* 44) [Convolution],(Unnamed Layer* 45) [Scale],(Unnamed Layer* 46) [ElementWise],(Unnamed Layer* 47) [Activation],(Unnamed Layer* 48) [Convolution],(Unnamed Layer* 49) [Scale],(Unnamed Layer* 50) [Activation],(Unnamed Layer* 51) [Convolution],(Unnamed Layer* 52) [Scale],(Unnamed Layer* 53) [ElementWise],(Unnamed Layer* 54) [Activation],(Unnamed Layer* 55) [Convolution],(Unnamed Layer* 56) [Scale],(Unnamed Layer* 57) [Activation],(Unnamed Layer* 58) [Convolution],(Unnamed Layer* 59) [Scale],(Unnamed Layer* 60) [Convolution],(Unnamed Layer* 61) [Scale],(Unnamed Layer* 62) [ElementWise],(Unnamed Layer* 63) [Activation],(Unnamed Layer* 64) [Convolution],(Unnamed Layer* 65) [Scale],(Unnamed Layer* 66) [Activation],(Unnamed Layer* 67) [Convolution],(Unnamed Layer* 68) [Scale],(Unnamed Layer* 69) [ElementWise],(Unnamed Layer* 70) [Activation],(Unnamed Layer* 71) [Convolution],(Unnamed Layer* 72) [Scale],(Unnamed Layer* 73) [Activation],(Unnamed Layer* 74) [Convolution],(Unnamed Layer* 75) [Scale],(Unnamed Layer* 76) [ElementWise],(Unnamed Layer* 77) [Activation],(Unnamed Layer* 78) [Convolution],(Unnamed Layer* 79) [Scale],(Unnamed Layer* 80) [Activation],(Unnamed Layer* 81) [Convolution],(Unnamed Layer* 82) [Scale],(Unnamed Layer* 83) [ElementWise],(Unnamed Layer* 84) [Activation],(Unnamed Layer* 85) [Convolution],(Unnamed Layer* 86) [Scale],(Unnamed Layer* 87) [Activation],(Unnamed Layer* 88) [Convolution],(Unnamed Layer* 89) [Scale],(Unnamed Layer* 90) [ElementWise],(Unnamed Layer* 91) [Activation],(Unnamed Layer* 92) [Convolution],(Unnamed Layer* 93) [Scale],(Unnamed Layer* 94) [Activation],(Unnamed Layer* 95) [Convolution],(Unnamed Layer* 96) [Scale],(Unnamed Layer* 97) [ElementWise],(Unnamed Layer* 98) [Activation],(Unnamed Layer* 99) [Convolution],(Unnamed Layer* 100) [Activation],(Unnamed Layer* 101) [Convolution],(Unnamed Layer* 102) [Activation],(Unnamed Layer* 103) [Convolution],(Unnamed Layer* 104) [Activation],(Unnamed Layer* 105) [Convolution],(Unnamed Layer* 106) [Activation],(Unnamed Layer* 107) [Convolution],(Unnamed Layer* 108) [Activation],(Unnamed Layer* 109) [Convolution],(Unnamed Layer* 110) [Activation],(Unnamed Layer* 111) [Convolution],(Unnamed Layer* 112) [Activation],(Unnamed Layer* 113) [Convolution],(Unnamed Layer* 114) [Activation],(Unnamed Layer* 115) [Convolution],(Unnamed Layer* 116) [Activation],(Unnamed Layer* 117) [Convolution],(Unnamed Layer* 118) [Activation]},
[TensorRT] INFO: --------------- Layers running on GPU:
[TensorRT] INFO: (Unnamed Layer* 131) [Constant], (Unnamed Layer* 119) [Convolution], (Unnamed Layer* 120) [Convolution], (Unnamed Layer* 121) [Convolution], (Unnamed Layer* 122) [Convolution], (Unnamed Layer* 123) [Convolution], (Unnamed Layer* 124) [Convolution], (Unnamed Layer* 125) [Convolution], (Unnamed Layer* 126) [Convolution], (Unnamed Layer* 127) [Convolution], (Unnamed Layer* 128) [Convolution], (Unnamed Layer* 129) [Convolution], (Unnamed Layer* 130) [Convolution], (Unnamed Layer* 132) [PluginV2IOExt],
[TensorRT] ERROR: ../builder/cudnnBuilderUtils.cpp (427) - Cuda Error in findFastestTactic: 700 (an illegal memory access was encountered)
[TensorRT] ERROR: ../rtSafe/safeRuntime.cpp (32) - Cuda Error in free: 700 (an illegal memory access was encountered)
terminate called after throwing an instance of 'nvinfer1::CudaError'
  what():  std::exception

**Comment by nvpohanh** [comment]
Which hardware and software stacks did you run this on? Also, are you planning to submit this result to v0.7 submission?
